tool/test-bundled-gems.rb

```diff
@@ -33,7 +34,7 @@
   next if bundled_gems&.none? {|pat| File.fnmatch?(pat, gem)}
   next unless File.directory?("#{gem_dir}/src/#{gem}/test")
 
-  test_command = [ruby, "-C", "#{gem_dir}/src/#{gem}", rake, "test"]
+  test_command = [ruby, run_opts, "-C", "#{gem_dir}/src/#{gem}", rake, "test"].compact
```

Comment:
`RUN_OPTS` generally accepts multiple options (but this doesn't).

How about (adding `require 'shellwords'` and) using `run_opts = ENV["RUN_OPTS"]&.shellsplit` and changing it to `*run_opts` here?

Reply:
Sounds good. Updated 👍